Formula/e/excalidraw-converter.rb

```diff
@@ -6,6 +6,11 @@ class ExcalidrawConverter < Formula
   license "MIT"
   head "https://github.com/sindrel/excalidraw-converter.git", branch: "master"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+\.\d+\.\d+)$/i)
+  end
+
```

Comment:
does the default livecheck work?

Reply:
I want to make sure it only triggers on releases, I don't want release candidates etc. Is the default livecheck behavior to strictly check for "vX.X.X"? I could not find any information in the docs. Also the use of this regex is very common.

Reply:
yeah, it is common because there are times that it would capture the rc, beta releases, but for people who strictly follow semantic versioning, there is no need to add the livecheck part.

Reply:
Ah, okay. So if I strictly follow semantic versioning, this is the default livecheck behavior?

`v1.0.0` <- captured
`v1.0.1-rc0` <- not captured

Or do I still need the regex to make sure beta or rc releases are ommitted?

Reply:
the default livecheck just works fine

```
/o/h/L/T/h/homebrew-core (master) > brew livecheck --autobump -r excalidraw-converter
excalidraw-converter: 1.4.5 ==> 1.4.5
```

Reply:
Fixed in latest revision

Reply:
thank you!